.gitignore

```diff
@@ -11,7 +11,7 @@
 # production
 /build
 /staging
-/prod
```

Comment:
i'm using these folders to manage our buckets, so i don't wanna accidentally push them :P